### PR TITLE
F distribution

### PR DESCRIPTION
**Pull request by YoshikawaMasashi**
split from #1177

### CODE REVIEW

**Comment by YoshikawaMasashi** [comment]
jenkins, test this please.

**Comment by toslunar** [comment]
I'll review this after #1414 is merged (and the conflicts are resolved).

File: tests/cupy_tests/random_tests/test_generator.py [hunk 1]
```diff
@@ -133,7 +133,6 @@ class TestBinomial(RandomGeneratorTestCase):
     def test_binomial(self):
         self.generate(n=self.n, p=self.p, size=(3, 2))
 
-
```
Comment (toslunar) [inline tests/cupy_tests/random_tests/test_generator.py]: This blank line is needed to follow the coding style.

**Comment by YoshikawaMasashi** [comment]
jenkins, test this please.

**Pull request merged**